blogposts/2021/release-3.30.md

```diff
@@ -0,0 +1,204 @@
+---
+title: "Sourcegraph 3.30 release"
+publishDate: 2021-07-20T10:00-07:00
+description: "The Sourcegraph 3.30 release introduces"
+tags: [blog, release]
+slug: "release/3.30"
+published: false
+changelogItems:
+  - description: "Added support for `select:file.directory` in search queries, which returns unique directory paths for results that satisfy the query."
+    url: https://github.com/sourcegraph/sourcegraph/pull/22449
+    category: Search
+  - description: "An `sg_service` Postgres role has been introduced, as well as an `sg_repo_access_policy` policy on the `repo` table that restricts access to that role. The role that owns the `repo` table will continue to get unrestricted access."
+    url: https://github.com/sourcegraph/sourcegraph/pull/22303
+    category: Repositories
```

Comment:
Can cut this. 

(Side note @abschu - it's helpful with your comments of cutting sections to highlight all 3 lines and delete them in the suggestion the way I've done here. Let me know if I can show you how to do this!)

```suggestion
```